### PR TITLE
MONIT-33770: Proxy Ingest w/o Admin Auth

### PR DESCRIPTION
**Pull request by deosu**
Please check the acceptance criteria of [MONIT-33770](https://jira.eng.vmware.com/browse/MONIT-33770)

### CODE REVIEW

File: proxy/src/main/java/com/wavefront/agent/listeners/WavefrontPortUnificationHandler.java [hunk 1]
```diff
@@ -211,6 +221,16 @@ public WavefrontPortUnificationHandler(
                     Metrics.newCounter(
                         new TaggedMetricName(
                             "logs." + handle, "discarded", "format", format.name().toLowerCase())));
+    this.discardedLogsMissingLogServerInfoCounter =
```
Comment (amitw-vmware) [inline proxy/src/main/java/com/wavefront/agent/listeners/WavefrontPortUnificationHandler.java:224]: good troubleshooting!

**Review by amitw-vmware — APPROVED** [review]
lgtm

**Review by amitw-vmware — APPROVED** [review]
lgtm

**Review by amitw-vmware — APPROVED** [review]
lgtm

**Review by joannak-vmware — APPROVED** [review]
LGTM

**Pull request merged**